src/operations/allow_trust.js

```diff
@@ -30,7 +30,13 @@ export function allowTrust(opts) {
   } else {
     throw new Error('Asset code must be 12 characters at max.');
   }
-  attributes.authorize = opts.authorize;
+
+  if (typeof opts.authorize === 'boolean') {
+    attributes.authorize = +opts.authorize;
```

Comment:
I think it would be better to reference the xdr constant

Reply:
this particular operation is casting the value from boolean to int. People can still refer to the xdr constant in their code.

Reply:
@abuiles there is an implicit assumption that the authorized flag is 1. I think it would be better rewrite the code as:

```javascript
  if (typeof opts.authorize === 'boolean') {
    if (opts.authorize) {
        attributes.authorize = xdr.TrustLineFlags.authorizedFlag;
    } else {
        attributes.authorize = 0;
    }
  } else {
    attributes.authorize = opts.authorize;
  }
```